manifests/o/OneScript/OneScript/1.8.0/OneScript.OneScript.locale.ru-RU.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate
+
+PackageIdentifier: OneScript.OneScript
+PackageVersion: 1.8.0
+PackageLocale: ru-RU
+Publisher: 1Script Team (Open Source)
+PackageName: OneScript
```

```suggestion
PackageName: OneScript engine
```
This must match Control Panel.